src/viur/core/__init__.py

```diff
@@ -1,27 +1,8 @@
 """
-                 iii
-                iii
-               iii
+ViUR-core
+Copyright © 2024 Mausbrand Informationssysteme GmbH
 
```

Comment:
Maybe it would be not bad to link the docs:
```suggestion

https://core.docs.viur.dev
```